### ISSUE TITLE
Missing Segment.MatchType in export

### ISSUE THREAD

**Issue by kevin-ip** [markphelps/flipt#710]
**Describe the bug**
Exporting a segment with match type = "Match Any" and import it back updates the match type to "Match All"

**Version Info**
Run `flipt --version` and paste the output here.
```
/ $ ./flipt -v

 _____ _ _       _
|  ___| (_)_ __ | |_
| |_  | | | '_ \| __|
|  _| | | | |_) | |_
|_|   |_|_| .__/ \__|
          |_|

Version: 1.6.1
Commit: 8ea0d0a8c150305e72b079f44a6b01d74b917bc0
Build Date: 2022-02-18T16:55:17Z
Go Version: go1.17.6
/ $
```

**To Reproduce**
Steps to reproduce the behavior.
1. start container with the following docker-compose.yaml
```
version: "3.9"

services:
  flipt:
    image: markphelps/flipt:v1.6.1
    ports:
      - 8080:8080
    volumes:
      - ${PWD}/settings:/opt/settings
```
2. visit localhost:8080
3. create a segment with Match Type = "Match Any"
4. create two constraints
5. logon to the container: `docker exec -it $containerID sh`
6. export the settings: `./flipt export -o /opt/settings/out.yaml`
7. go back to the host terminal
8. shutdown and restart the docker compose: `docker-compose down && docker-compose up -d`
9. logon to the container: `docker exec -it $containerID2 sh`
10. import the settings: `./flipt import /opt/settings/out.yaml`
11. visit localhost:8080 and see the segment
13. the Match Type is updated to "Match All"

**Expected behavior**
Expected the match type of the imported segment be "Match Any"

**Screenshots**

**Additional context**

Examples:

* OS
* Config file used
* Database used (SQLite or Postgres)
SQLite

**Comment by markphelps** [comment]
Thanks for reporting @kevin-ip ! I've got a fix PR for this

**Comment by markphelps** [comment]
@kevin-ip this should be fixed in [v1.6.2](https://github.com/markphelps/flipt/releases/tag/v1.6.2)

**Comment by kevin-ip** [comment]
Awesome! Thanks for the quick fix.
